### ISSUE TITLE
Always-underlined href links in blog posts

### ISSUE THREAD

**Issue by PeterBreen** [DemocracyLab/CivicTechExchange#964]
The accessibility designers also asked for a small change to our Latest Blog Posts module on the site homepage - rather than the underline indicating a hyperlink appearing on hover, have it render all the time. It should be fairly straightforward, something like this will probably work:

```css
.LatestBlogPosts-excerpt a { 
  text-decoration: underline;
}
```

File location:
https://github.com/DemocracyLab/CivicTechExchange/blob/master/civictechprojects/static/css/partials/_LatestBlogPosts.scss#L47

**Comment by SleeplessRegina** [comment]
Underlined href links in blog posts, make links more straightforward and clear.
